content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -210,6 +210,31 @@ kubectl apply -f my-private-reg-pod.yaml
 kubectl get pod private-reg
 ```
 
+{{< note >}}
+In case the the Pod fails to start with the status `ImagePullBackOff`, view the Pod events:
```

Comment:
nit: can you remove the extra `the` in the first sentence?

Reply:
🤦🏻 fixed, thanks @kbhawkey